### PR TITLE
fix(core): [lw-11146] fix circular deps in core package

### PR DESCRIPTION
**Pull request by vetalcore**
# Checklist

- [x] JIRA - [LW-11146](https://input-output.atlassian.net/browse/LW-11146)
- [ ] Proper tests implemented
- [ ] Screenshots added.

---

## Proposed solution

Explain how does this PR solves the problem stated in JIRA ticket.
You can also enumerate different alternatives considered while approaching this task.

## Testing

Describe here, how the new implementation can be tested.
Provide link or briefly describe User Acceptance Criteria/Tests that need to be met

## Screenshots

Attach screenshots here if implementation involves some UI changes

[LW-11146]: https://input-output.atlassian.net/browse/LW-11146?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: packages/core/src/shared-wallets/utils/has-signed.ts [hunk 1]
```diff
@@ -1,5 +1,5 @@
 import { Wallet } from '@lace/cardano';
-import { paymentScriptKeyPath, stakingScriptKeyPath } from '@src/shared-wallets';
+import { paymentScriptKeyPath, stakingScriptKeyPath } from '@src/shared-wallets/utils/derivation-path';
```
Comment (szymonmaslowski) [inline packages/core/src/shared-wallets/utils/has-signed.ts:2]:
```suggestion
import { paymentScriptKeyPath, stakingScriptKeyPath } from './derivation-path';
```

**Review by szymonmaslowski — APPROVED** [review]
Left a suggestion

**Review by danielmain — APPROVED** [review]

**Comment by pczeglik-iohk** [comment]
# Allure Report
[`allure-report-publisher`](https://github.com/andrcuns/allure-report-publisher) generated test report!



**processReports**: ✅ [test report](https://lace-qa:8PP5wBaDV6UoXj@dq4ajm5i5q7bz.cloudfront.net/smoke/linux/chrome/6519/10216773405/index.html) for [0554fb4b](https://github.com/input-output-hk/lace/pull/1342/commits/0554fb4b025064953fedc981d3a41f843f359c5e)
|       | passed | failed | skipped | flaky | total | result |
|-------|--------|--------|---------|-------|-------|--------|
| Total | 32     | 0      | 0       | 0     | 32    | ✅     |

**Review by shawnbusuttil — APPROVED** [review]

**Comment by emiride** [comment]
I tested this and it works as expected.

**Pull request merged**